kale/codegen/generate_code.py

```diff
@@ -135,19 +135,21 @@ def generate_lightweight_component(template, step_name, step_data, nb_path,
     step_marshal_out = step_data['outs']
 
     # TODO: Remove some parameters and pass them with **metadata
-    return template.render(
+    fn_code = template.render(
         step_name=step_name,
         function_body=[step_source],
         in_variables=step_marshal_in,
         out_variables=step_marshal_out,
         nb_path=nb_path,
         **metadata
     )
+    # fix code style using pep8 guidelines
+    return autopep8.fix_code(fn_code)
```

Comment:
nit [but no strong opinion]:
```python
    return autopep8.fix_code(template.render(step_name=step_name,               
                                             function_body=[step_source],       
                                             in_variables=step_marshal_in,      
                                             out_variables=step_marshal_out,    
                                             nb_path=nb_path,                   
                                             **metadata)
```

Reply:
I'd rather keep it as it is. I think it is more readable, it does not make much sense to oversimplify in this case